From 56f22b7e16006a4201ed4cf9d21e5ec26faea638 Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Tue, 10 Jan 2012 15:17:39 -0500
Subject: [PATCH 07/63] pcmenc: set frame_size to 0.

This indicates that the actual frame size is based on the buf_size passed to
avcodec_encode_audio().
---
 libavcodec/pcm.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libavcodec/pcm.c b/libavcodec/pcm.c
index 76d5c10..3223112 100644
--- a/libavcodec/pcm.c
+++ b/libavcodec/pcm.c
@@ -33,7 +33,7 @@
 
 static av_cold int pcm_encode_init(AVCodecContext *avctx)
 {
-    avctx->frame_size = 1;
+    avctx->frame_size = 0;
     switch(avctx->codec->id) {
     case CODEC_ID_PCM_ALAW:
         pcm_alaw_tableinit();
-- 
1.7.5.4

